### ISSUE TITLE
Wrong default value in description of `coverageAnalysis` option in source code on npmjs.com

### ISSUE THREAD

**Issue by Userbit** [stryker-mutator/stryker#1888]
For current v2.3.0 version of "@stryker-mutator/core" on npmjs.com inside of package source code  in [@stryker-mutator/core/src/StrykerCli.js](https://source.runkitcdn.com/npm/@stryker-mutator/core/src/StrykerCli.js?engine=10.x.x&t=1574608746265) on 36th line specified the following: 

```javascript
.option('--coverageAnalysis <perTest|all|off>', 'The coverage analysis strategy you want to use. Default value: "perTest"')
```

But in fact the valid default value of `coverageAnalysis` option is `off`. 
I don't understand as to fix this issue in the GitHub repo. 
Because results of investigation established that is all correctly in corresponding *.ts files.

**Comment by simondel** [comment]
If I remove the setting from my config, I get the logging:
```
20:42:17 (15520) DEBUG TestFrameworkOrchestrator The `coverageAnalysis` setting is "off", not hooking into the test framework to achieve performance benefits.
```

So at runtime we get the setting 'off' but for some reason, the StrykerCli disagrees...

**Comment by simondel** [comment]
nvm. This is fixed in 2.4.0 because of https://github.com/stryker-mutator/stryker/commit/fbb8102dc890521a80ef1f87236193e506ab9922#diff-533cd7dc486614a3081f9891091442e7
